src/chocolatey/infrastructure.app/commands/ChocolateyInfoCommand.cs

```diff
@@ -83,6 +85,24 @@ public override void handle_additional_argument_parsing(IList<string> unparsedAr
             configuration.ListCommand.Exact = true;
         }
 
+        public override void handle_validation(ChocolateyConfiguration configuration)
+        {
+            base.handle_validation(configuration);
+
+            if (string.IsNullOrWhiteSpace(configuration.Input))
+            {
+                throw new ApplicationException(@"A single package id is required to run the info again.");
+            }
+
+            //Validate only a single package has been passed to info
+            //TODO: Provide ability to get info on a list of packages

```

Comment:
If we're going to add a TODO here, could we also open an issue and tag it in this TODO comment.

We may want to note in that issue that this may be better handled in `ChocolateyListCommand.cs` as info just overloads the List Command and sets `Exact` and `Verbose`. (This does bring up the question of if this validation should be there when `Exact` is given)